### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by cybercuisine**
Fixed typo in some documents.

### CODE REVIEW

**Comment by juliusvonkohout** [comment]
Only @thesuperzapper and @kimwnasptd can approve workflows.

**Comment by thesuperzapper** [comment]
@juliusvonkohout, Actually no one can approve workflows until this PR is merged that gives us write access:
- https://github.com/kubeflow/internal-acls/pull/618

That is also aside the fact that these kinds of typo-only PRs are always a little bit suspicious.

**Review by TheLoGgI — APPROVED** [review]
Minor changes, I approve

**Pull request closed without merging**
